tests/pricing_models/test_calc_in_given_out.py

```diff
@@ -2,6 +2,7 @@
 Testing for the calc_in_given_out of the pricing models.
 """
 
+# pylint: disable=too-many-lines
```

Comment:
looks like you can delete this line -- Matt added the pylint disable in his own PR.